### DISCUSSION TITLE
Adjusting menu bar height

### DISCUSSION THREAD

**Discussion by Mihokas99** [jordanbaird/Ice#129] [General]
I've been looking for app that I can use to adjust menu bar height but without any success. Ice and Bartender are the closest thing I have seen but neither are providing this option. Maybe this could be something to think about because thicker menu bar is a thing with newer macbooks  and Ice already provides other options to customize menu bar looks.

Edit: I just asked this from ChatGPT and it stated that this is not possible because Apple doesn't provide API to adjust Mac OS system elements. I just wonder how is it then possible to separate left and right part of menu bar and fade out the middle part.

**Comment by jordanbaird** [comment]
ChatGPT is correct. It unfortunately is not possible. Ice (and Bartender) achieves the menu bar split by essentially adding a layer on top of the menu bar. So the system element isn't actually being changed.
